exercises/practice/palindrome-products/palindrome_products_test.rb

```diff
@@ -2,94 +2,94 @@
 require_relative 'palindrome_products'
 
 class PalindromesTest < Minitest::Test
-  def test_find_the_smallest_palindrome_from_single_digit_factors
+  def test_for_the_smallest_palindrome_from_single_digit_factors
     # skip
     palindromes = Palindromes.new(max_factor: 9)
     palindromes.generate
-    smallest = palindromes.smallest
-    assert_equal 1, smallest.value
-    assert_equal [[1, 1]], smallest.factors
+    actual = palindromes.smallest.value, palindromes.smallest.factors
+    expected = 1, [[1, 1]]
+    assert_equal expected, actual
   end
 
-  def test_find_the_largest_palindrome_from_single_digit_factors
+  def test_for_the_largest_palindrome_from_single_digit_factors
     skip
     palindromes = Palindromes.new(max_factor: 9)
     palindromes.generate
-    largest = palindromes.largest
-    assert_equal 9, largest.value
-    assert_includes [[[1, 9], [3, 3]], [[3, 3], [1, 9]]], largest.factors
+    actual = palindromes.largest.value, palindromes.largest.factors
+    expected = [[9, [[1, 9], [3, 3]]], [9, [[3, 3], [1, 9]]]]
```

Comment:
Likely one extra pair of `[]` brackets... Gotta go back and look at how this is created.  But also, perhaps not really problematic.

Just knowing that the assignment itself will be evaluated as an Array is good information to have in mind as a Ruby developer though.

We should avoid nesting to the extent that we can though, as it can be confusing as a learner.  (It goes the other way, though, if you are unaware.)